frontend/src/routes/Governance/discovered/ByCluster/common.tsx

```diff
@@ -23,8 +23,8 @@ export const policyViolationSummary = (discoveredPolicyItems: DiscoveredPolicyIt
   let unknown = 0
   for (const policy of discoveredPolicyItems) {
     let compliance: string
-
-    if (policy.apigroup === 'constraints.gatekeeper.sh') {
+    // Kyverno resources also use the totalViolations field
+    if (['constraints.gatekeeper.sh', 'kyverno.io'].includes(policy.apigroup)) {
       compliance = getConstraintCompliance(policy?.totalViolations)
```

Comment:
Could you please rename `getConstraintCompliance` since it implies only Gatekeeper constraints use it?